x-pack/plugins/security/public/management/role_mappings/role_mappings_grid/role_mappings_grid_page.tsx

```diff
@@ -413,7 +413,7 @@ export class RoleMappingsGridPage extends Component<Props, State> {
                                 'xpack.security.management.roleMappings.actionDeleteAriaLabel',
                                 {
                                   defaultMessage: `Delete '{name}'`,
-                                  values: { name },
+                                  values: { name: record.name },
```

Comment:
WUT, it seems it didn't complain before as it was falling back to `windows.name` :see_no_evil: But I thought `no-implicit-globals` should have forbidden that? Anyway, thanks for fixing this!